generated/nixnet/nixnet_service.cpp

```diff
@@ -169,7 +169,7 @@ namespace nixnet_grpc {
       auto session_ref_grpc_session = request->session_ref();
       nxSessionRef_t session_ref = session_repository_->access_session(session_ref_grpc_session.id(), session_ref_grpc_session.name());
       u8* value_buffer = (u8*)request->value_buffer().c_str();
-      u32 size_of_value_buffer = static_cast<u32>(request->value_buffer().size());
+      u32 size_of_value_buffer = static_cast<u32>(request->value_buffer().size() * sizeof(u8));
```

Comment:
Isnt sizeof(u8) 1 byte only? but yeah this is also correct.

Reply:
Yes, that's correct. But this is auto-generated code and I had tagged the API in metadata to call out that this parameter expects length in bytes. This auto generated change came in this PR because of merge conflicts. I will add a note here so that owners won't get confused about the change.